webcompat/static/js/lib/labels.js

```diff
@@ -24,9 +24,9 @@ issues.LabelsView = Backbone.View.extend({
     "click .js-LabelEditorLauncher.is-active": "closeEditor"
   },
   keyboardEvents: {
-    "l": "labelWrap"
+    "l": "labelWarp"
```

Comment:
Ah, now I get `labelWrap`, it was a typo for `labelWarp`. :)))

So, the name `githubWarp` was a silly name playing off the mario brothers concept of using a pipe to warp to another level:

![](http://www.videogamesblogger.com/wp-content/uploads/2010/04/super-mario-bros-1-warp-zone-walkthrough-screenshot-small.jpg) (hence the dumb `warpPipe` variable name...)

So I'm not sure `labelWarp` is meaningful. What we're doing is opening the label editor.